midi_app_controller/gui.py

```diff
@@ -0,0 +1,426 @@
+import sys
+from pathlib import Path
+from app_model.types import Action
+import yaml
+from PyQt5.QtWidgets import QApplication, QWidget, QVBoxLayout, QPushButton, QLabel, QMainWindow, QAction, QHBoxLayout, \
+    QDial, QComboBox, QFormLayout, QMenuBar
+from midi_app_controller.actions.bound_controller import BoundController, ButtonActions, KnobActions
+from midi_app_controller.models.binds import ButtonBind, KnobBind, Binds
+
+
+class MIDIControllerView(QMainWindow):
+    def __init__(self):
+        super().__init__()
+        self.setupUI()
+        self.loadSettings()
+
+    def ui_action_to_model_action(self, action_name: str):
+        """
+        Converts an action name to a model Action.
+
+        Parameters
+        ----------
+        action_name: str
+            Name of the action.
+
+        Returns
+        -------
+        Action
+            An instance of Action with the given name.
+        """
+
+        if action_name == 'None':
+            return None
+        return Action(id=action_name, callback="", title=action_name)
+
+    def setupUI(self):
+        """
+        Sets up the UI.
```

Comment:
One line docstrings should be in one line (this is a convention used also by napari).